Sources/TensorFlow/Operators/Math.swift

```diff
@@ -1014,6 +1014,23 @@ func _vjpElu<T: TensorFlowFloatingPoint>(
     return (y, { v in Raw.eluGrad(gradients: v, outputs: y) })
 }
 
+/// Computes `leakyRelu` of the specified tensor element-wise.
+/// Specifically, computes `max(features, features * alpha)`.
+@inlinable
+@differentiable(wrt: x, vjp: _vjpLeakyRelu)
+public func leakyRelu<T: TensorFlowFloatingPoint>(_ x: Tensor<T>, alpha: Double = 0.2) -> Tensor<T> {
```

Comment:
- For consistency with tf.keras, `alpha` should be `0.3`.
- Fit within 100 columns.

Reply:
@rxwei so 0.2 is the default in the raw c++ apis tho, what would you like to choose?

Reply:
Ok, in that case, `0.2` is good. Thanks!